routeros/provider_schema_helpers.go

```diff
@@ -334,6 +334,14 @@ var (
 		Description: "Layer2 Maximum transmission unit. " +
 			"[See](https://wiki.mikrotik.com/wiki/Maximum_Transmission_Unit_on_RouterBoards).",
 	}
+	PropL2MtuRw = &schema.Schema{
+		Type:     schema.TypeInt,
+		Optional: true,
+		Description: "Layer2 Maximum transmission unit. " +
+			"[See](https://wiki.mikrotik.com/wiki/Maximum_Transmission_Unit_on_RouterBoards).",
+		ValidateFunc:     validation.IntBetween(0, 65536),
```

Comment:
Michael, greetings!
The range should be, I think, from 1 to 65535
```
		if v < min || v > max {
			errors = append(errors, fmt.Errorf("expected %s to be in the range (%d - %d), got %d", k, min, max, v)))
			return warnings, errors
		}
```

Reply:
In the documentation, it [says](https://help.mikrotik.com/docs/display/ROS/Ethernet#Ethernet-Properties) `[0..65536]`. This seems like inclusive 0 and 65536, but that doesn't make any sense. 

Anyway, they have their own validation per each resource, and, I guess, also device-specific. So, I've updated my PR.